src/components/ui-shell/ui-shell.config.js

```diff
@@ -135,6 +151,7 @@ module.exports = {
   },
   context: {
     header,
+    headernav,
```

Comment:
Cannot see `headernav` definition, which seems to have caused the Travis error:

```
[19:21:15] TypeError: Cannot read property 'forEach' of undefined
    at cache.get.then (/home/travis/build/IBM/carbon-components/tools/templates.js:100:21)
    at <anonymous>:null:null
```

The underlying `fractal.load()` fails to populate `componentSource` property in such case, and thus causes JavaScript error at https://github.com/IBM/carbon-components/blob/v9.22.0/tools/templates.js#L100.